tools/cabana/SConscript

```diff
@@ -39,7 +39,9 @@ if GetOption('extras'):
   cabana_env.Program('tests/test_cabana', ['tests/test_runner.cc', 'tests/test_cabana.cc', cabana_lib], LIBS=[cabana_libs])
 
 output_json_file = 'tools/cabana/dbc/car_fingerprint_to_dbc.json'
-generate_dbc = cabana_env.Command('#' + output_json_file,
-                                  ['dbc/generate_dbc_json.py'],
+
+os.system('git log -n1 --format=format:"%H" > .generate_dbc_dependencies')
+generate_dbc = cabana_env.Command(['#' + output_json_file],
+                                  ['dbc/generate_dbc_json.py', '.generate_dbc_dependencies'],
                                   "python3 tools/cabana/dbc/generate_dbc_json.py --out " + output_json_file)
 cabana_env.Depends(generate_dbc, ["#common", "#selfdrive/boardd", "#opendbc", "#cereal"])
```

Comment:
```suggestion
cabana_env.Depends(generate_dbc, ["#common", "#selfdrive/boardd", Glob("#opendbc/*"), "#cereal"])
```

What about something like this?

Reply:
sonsc will iterate through the contents of all files and generate hash values. the performance can be much slower than check the  latest commit

Reply:
What about *.dbc? If I rename/add/remove DBC file it should probably update